modules/minimum-required-permissions-upi-azure.adoc

```diff
@@ -0,0 +1,234 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+
+[id="minimum-required-azure-permissions_{context}"]
+= Required Azure permissions in user-provisioned infrastructure
+
+When you assign the contributor and user access administrator roles to the service principal, you automatically grant all the required permissions. To deploy and destroy an {product-title} cluster with installer-provisioned infrastructure, the service principal requires the following permissions.
+
+Following are the permissions required for creating the {product-title} cluster.
+
+.Required permissions for creating authorization resources
+[%collapsible]
+====
+* `Microsoft.Authorization/policies/audit/action`
+* `Microsoft.Authorization/policies/auditIfNotExists/action`
+* `Microsoft.Authorization/roleAssignments/read`
+* `Microsoft.Authorization/roleAssignments/write`
+====
+
+.Required permissions for creating compute resources
```

Comment:
3 permissions should be moved here from optional, as suggested in other comments.